recipe/meta.yaml

```
@@ -37,33 +31,35 @@ requirements:
   build:
     - python
     # run-dep as a workaround for now; upstream package still imports from distutils
-    - setuptools <74  # [py>311]
+    - setuptools <74
+    - {{ compiler('c') }}
     - {{ posix }}bison
     - {{ posix }}flex
     - {{ posix }}patch
     - {{ posix }}sed
     - meson >=0.60.0
     - ninja-base
     - pkg-config
-    - {{ compiler('c') }}
   host:
-    - pkg-config
     - python
     # X11/XCB dependencies through cairo.
     - cairo {{ cairo }}
-    - glib {{ glib }}
+    - glib >=2.82,<2.85
     - libffi {{ libffi }}
     - pthread-stubs 0.3
     - zlib {{ zlib }}  # [win]
-    - setuptools <74
+    - packaging
+    # ../meson.build:29:8: ERROR: $PREFIX/bin/python is missing modules: setuptools
+    - setuptools 72.1.0
   run:
     - python
     # through run_exports
     - libffi
-    - libglib
+    - libglib >=2.82,<2.85
+    - glib >=2.82,<2.85
```

```suggestion
    - libglib >=2.82.0,<2.85.0
    - glib >=2.82.0,<2.85.0
```
I saw the same pairing of glib/libglib in the opencv package, see discussion https://github.com/AnacondaRecipes/opencv-feedstock/pull/38/files#r2152348642. 

Should both packages be at runtime?